src/registrar/migrations/0006_alter_domainapplication_federal_agency.py

```diff
@@ -0,0 +1,20 @@
+# Generated by Django 4.1.3 on 2022-12-12 14:48
```

Comment:
**question (non-bloking)**: I'm wondering if there is a merge conflict here. Did this migration already get made?

Reply:
I am not sure about any merge conflict. I looked on `main` and didn't see any `0005` or `0006`.

I do see that I've included two migration files on this PR and there really only needed to be one, so I've condensed them.